redis-test/test_strings.py

```diff
@@ -113,3 +113,18 @@ def test_select():
         raise Exception('Expect that `%s` does not work' % query)
     except redis.exceptions.ResponseError as ex:
         assert str(ex) == 'invalid DB index'
+
+def test_redis_default_database_count():
+    response = requests.get('http://127.0.0.1:10000/v2/config/redis_default_database_count')
```

Comment:
Again, this won't work against regular Redis :-(

Reply:
fixed.